modules/acquia_cms_headless/composer.json

```diff
@@ -69,7 +69,8 @@
                 "3111456 - Unable to resolve path on node in other language than default": "https://www.drupal.org/files/issues/2022-11-30/decouple_router-3111456-resolve-language-issue-58.patch"
             },
             "drupal/subrequests": {
-                "Get same results on different request": "https://www.drupal.org/files/issues/2019-07-18/change_request_type-63049395-09.patch"
+                "3049395 - Page Cache causes different subrequests to return the same responses": "https://www.drupal.org/files/issues/2019-07-18/change_request_type-63049395-09.patch",
+                "3338312 - Incompatible with latest versions of symfony/http-foundation": "https://git.drupalcode.org/project/subrequests/-/merge_requests/14.patch"
```

Comment:
@lauriii benefit of using MR for patch is whenever file get changed in new releases patch needs to be updated followed by ACMS Headless release. To avoid that MR is best solution anyone can go and update the MR accordingly as per new release and it starts working.

Reply:
That's fair. I still think we shouldn't rely on MR patches because it introduces a serious security vulnerability. This is caused by the fact that _anyone_ is able to push _any_ code to that MR, which would then be loaded into Acquia CMS sites without any user intervention.

Reply:
sure do necessary update.